app/controllers/reviews_controller.rb

```diff
@@ -2,13 +2,10 @@ class ReviewsController < ApplicationController
   def create
     @review = Review.new(review_params)
     @show = Show.find(params[:show_id])
-    @user = User.find(current_user)
+    @user = User.find(current_user.id)
```

Comment:
Would this not be simpler to write as `@user = current_user`? Why are you hitting the database again?